src/pages/linux/how-to-create-a-scheduled-cron-job/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How to create a scheduled cron job
+---
+## What is cron? 
+
+Cron can best be described as a time-based job-scheduler for UNIX like systems. 
+
+It typically automates system maintenance or administration
+
+Read more about <a href='https://en.wikipedia.org/wiki/Cron' target='_blank' rel='nofollow'>CRON</a>
+
+
+## Create a basic cronjob
+
+When logged in to your shell, run the command `crontab -e`
+
+Typically there is a short explanation already in the crontab. 
+
+The syntax is written in the following order `# m h dom mon dow command`
+
+1. m is for minutes 0-59
+
+2. h is for hour 0-23
+
+3. is for day of month 0-31
```

Comment:
I think you forgot "dom" here, as well as the variables for the following list items.